src/window/basic_frame.rs

```diff
@@ -288,14 +288,12 @@ impl Frame for BasicFrame {
                         );
                         data.position = (surface_x, surface_y);
                         if resizable {
-                            change_pointer(&pointer, data.location, Some(serial));
+                            change_pointer(&pointer, data.location, Some(serial))
                         }
                     }
                     Event::Leave { serial, .. } => {
                         data.location = Location::None;
-                        if resizable {
-                            change_pointer(&pointer, data.location, Some(serial));
-                        }
+                        change_pointer(&pointer, data.location, Some(serial))
```

Comment:
Why was this `if` removed ?

Reply:
Ah, to be honest I think this is just a mistake I didn't catch in the git diff